src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -176,30 +171,6 @@ public abstract class BaseCycloneDxMojo extends AbstractMojo implements Contextu
     @Parameter(property = "excludeTypes", required = false)
     private String[] excludeTypes;
 
-    /**
-     * Excluded ArtifactIds.
-     */
-    @Parameter(property = "excludeArtifactId", required = false)
-    protected String[] excludeArtifactId;
-
-    /**
-     * Excluded GroupIds.
-     */
-    @Parameter(property = "excludeGroupId", required = false)
-    protected String[] excludeGroupId;
```

Comment:
This should continue to be in BaseCycloneDxMojo. Both makeBom and makeAggregateBom should account for exclusions.

Reply:
moved back to BaseCycloneDxMojo